test/knexfile.js

```diff
@@ -19,6 +19,16 @@ var pool = {
   },
 };
 
+var poolSqlite = {
+  min: 0,
+  max: 1,
+  acquireTimeoutMillis: 5000,
```

Comment:
If possible we could try to have subsecond timeouts in tests to prevent test runs to take too long just waiting around.

Reply:
Tried reducing it, but I think I was getting some flaky tests that way. Decreased it, let's see how it goes.